<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>httppostClass</class>
 <widget class="QMainWindow" name="httppostClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>httppost</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditUrl"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPost">
        <property name="text">
         <string>POST</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGet">
        <property name="text">
         <string>GET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidgetPostBody">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>4</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <widget class="QWidget" name="tabPostKeyValue">
        <attribute name="title">
         <string>x-www-form-urlencoded</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBoxAutoUCenterAuthentication">
           <property name="text">
            <string>Auto UCenter Authentication</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="url_encode_table_widget" name="tableWidgetUrlEncode">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <column>
            <property name="text">
             <string>Key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Post Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Auth</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value Type</string>
            </property>
            <property name="background">
             <color>
              <red>255</red>
              <green>170</green>
              <blue>0</blue>
             </color>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonInsert">
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRemove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLoad">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSave">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabFormData">
        <attribute name="title">
         <string>multipart/form-data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="form_data_table_widget" name="tableWidgetFormData">
           <column>
            <property name="text">
             <string>name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Content-Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Full File Path Name</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButtonFormInsertText">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Add Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormInsertImage">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Add Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormInsertRaw">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Add Raw</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormInsert">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormRemove">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormLoad">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonFormSave">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabJson">
        <attribute name="title">
         <string>application/json</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEditJson"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButtonJsonLoad">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonJsonSave">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabTextXml">
        <attribute name="title">
         <string>text/xml</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEditText"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonTextLoad">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonTextSave">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabHeaders">
        <attribute name="title">
         <string>Headers</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="headers_table_widget" name="tableWidgetHeaders">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <column>
            <property name="text">
             <string>Key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Post Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Auth</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButtonHeaderInsert">
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonHeaderRemove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonHeaderLoad">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonHeaderSave">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelResponse">
          <property name="text">
           <string>Response</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidgetResponse">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabResponseBody">
           <attribute name="title">
            <string>Content</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="plainTextEditResponseBody"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabResponseHeader">
           <attribute name="title">
            <string>Headers</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QTableWidget" name="tableWidgetResponseHeaders">
              <column>
               <property name="text">
                <string>Key</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Value</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>form_data_table_widget</class>
   <extends>QTableWidget</extends>
   <header>form_data_table_widget.h</header>
  </customwidget>
  <customwidget>
   <class>url_encode_table_widget</class>
   <extends>QTableWidget</extends>
   <header>url_encode_table_widget.h</header>
  </customwidget>
  <customwidget>
   <class>headers_table_widget</class>
   <extends>QTableWidget</extends>
   <header>headers_table_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
